spec/tree-view-package-spec.coffee

```diff
@@ -769,6 +769,39 @@ describe "TreeView", ->
           expect(child).not.toHaveClass 'expanded'
         expect(treeView.roots[0]).toHaveClass 'expanded'
 
+
+  describe "the alwaysCollapseRecursively config option", ->
+    it "defaults to unset", ->
+      expect(atom.config.get("tree-view.alwaysCollapseRecursively")).toBeFalsy()
+
+    describe "when an expanded directory is clicked", ->
```

Comment:
I would be more explicit here and add another `describe` noting that this only occurs when the setting is true.